vignettes/phyloVision.Rmd

```diff
@@ -43,28 +56,86 @@ library(ape)
 
 ## Creating a PhyloVision Object with a Tree
 
-Vision objects now support dendrograms for visualization and analysis. To create the PhyloVision object, you need all of the same data, an expression matrix, and metadata and/or signatures, but you also include a tree object from the Ape package.
+We'll proceed by reading in the raw expression matrix, tree, and meta data for a single clone (CP007) of approximately 600 cells from [Quinn et al, Science 2021](https://www.science.org/doi/abs/10.1126/science.abc1944).
 
-```{r create, eval=F}
-# Read the expression and meta data
-file_path <- "data/embryogenesis" # Replace the path
+```{r, eval=F}
+# Read in expression data
+expression <- read.table('data/metastasis/cp007_expression.tsv.gz', sep='\t',
```

Comment:
Changing to expression as name instead of expr is missing in the hotspot calls for nrow

Reply:
Fixed.